app/src/main/java/com/checkin/app/checkin/session/scheduled/ScheduledSessionCartView.kt

```diff
@@ -259,6 +262,10 @@ class ScheduledSessionCartView @JvmOverloads constructor(
         containerRemovePromo.visibility = View.VISIBLE
         tvAppliedPromoDetails.text = data.details
     }
+    private fun enablerewards(){
+        if (supportrewards)
```

Comment:
Technically you'll hide the view when supportrewards variable is false. Please fix this check. 
Second, please try and follow Java variable naming convention or atleast follow a pattern similar to the code in this file.